spotlight/src/main/java/com/takusemba/spotlight/SpotlightView.java

```diff
@@ -63,19 +63,27 @@ protected void onDraw(Canvas canvas) {
     }
 
     void startSpotlight(long duration, TimeInterpolator animation, AbstractAnimatorListener listener) {
-        ObjectAnimator objectAnimator = ObjectAnimator.ofFloat(this, "alpha", 0f, 1f);
-        objectAnimator.setDuration(duration);
-        objectAnimator.setInterpolator(animation);
-        objectAnimator.addListener(listener);
-        objectAnimator.start();
+        try{
+            ObjectAnimator objectAnimator = ObjectAnimator.ofFloat(this, "alpha", 0f, 1f);
+            objectAnimator.setDuration(duration);
+            objectAnimator.setInterpolator(animation);
+            objectAnimator.addListener(listener);
+            objectAnimator.start();
+        }catch(NullPointerException e){
```

Comment:
Because you null-checked at the diff of Spotlight.java, I think you don't need to try-catch here.
Thank you for the reporting, I will add a null check at `onAnimationEnd` in Spotlight.java

Reply:
I think so too, but I preferred not to risk it.